aggregated-dependency-score.go

```diff
@@ -74,7 +74,25 @@ func (eval *trustwhorthinessEvaluator) evaluate(ctx context.Context, p Package)
 	}
 
 	for _, dep := range deps {
-		tPrimeQ, err := eval.evaluate(ctx, dep)
+		// XXX sometimes different names can refer to the same package,
```

Comment:
is XXX a marker or TODO or WARN?

Reply:
I use it to indicate something that could be a problem, so I guess equivalent to WARN.